webapp/channels/src/components/user_settings/display/manage_languages/manage_languages.tsx

```diff
@@ -264,6 +278,7 @@ export class ManageLanguage extends React.PureComponent<Props, State> {
                 saving={this.state.isSaving}
                 inputs={[input]}
                 updateSection={this.props.updateSection}
+                disableEnterSubmit={true}
```

Comment:
I didn't realize we had this prop as well. That's definitely nicer than changing `SettingItemMax` since it's clearer about how this is supposed to behave